### ISSUE TITLE
 Question about COVID-19-PA-CXR-fused-dataset

### ISSUE THREAD

**Issue by Zoe1109SSS** [nspunn1993/COVID-19-PA-CXR-fused-dataset#1]
Hello, I have some questions about the data set. I observed that the training set, test set and verification set in this dataset did not classify the four diseases according to the labels, so I separated them according to the labels in the csv file, and then I found that the number of other pneumonia images was not consistent with that in the instructions you gave. Is there any impact? Or can you provide the original data set for the four conditions?

**Comment by Zoe1109SSS** [comment]
There are still some questions that need to be answered by you. Is there any particular reason why you separate the tuberculosis image from the other pneumonia images? In addition, the number of other pneumonia image files you provided in your data set is different from that mentioned in your paper. The actual number of images is 170 less than that mentioned in your paper. Could you please update the data?

**Comment by TsingZ0** [comment]
@Zoe1109SSS Hi, were you able to download the datasets? The link doesn’t seem to be working now. Do you happen to have an archived version?
